### ISSUE TITLE
Pfsense null configuration XML tag format has changed

### ISSUE THREAD

**Issue by redshiftzero** [freedomofpress/securedrop#2282]
## Description 

It looks like there have been some changes to the XML null configuration tag format used by pfSense, now requiring `<tag></tag>` instead of the `<tag/>` format currently used in the templates that SecureDrop is shipping. 

Upstream issue: https://redmine.pfsense.org/issues/6893

## Steps to Reproduce

1. Download firewall templates.
2. Attempt to load.

## Expected Behavior

Template loads obviating the need for manual config.

## Actual Behavior

The template does not load. 

## Comments

This necessitates the templates to be regenerated using the (latest) updated format. 

Given the tight timeline until 0.4.3 is released, I suggest that we push the resolution of this into 0.4.4 to give sufficient time to generate and test the updated firewall templates. In the meantime, people installing SecureDrop will need to fall back to the manual configuration, and the documentation should indicate that manual configuration is necessary to avoid frustration in current users. 

@freddymartinez9 discovered this while testing #2281 on hardware.

**Comment by freddymartinez9** [comment]
I realized yesterday that I am in the middle of a fresh install with the latest pfSense. When I have a SD install complete with a clean (factory reset device pfSense) I can do an export of their configurations, so we won't have to manually fix these tags.

**Comment by freddymartinez9** [comment]
pushed this commit https://github.com/freedomofpress/securedrop/commit/ac4ce01382bd23a156140ba1d3700f7171292a77 but it requires a fresh pfSense firewall reset.  I will try to test this week

**Comment by freddymartinez9** [comment]
I tested my commit https://github.com/freedomofpress/securedrop/commit/ac4ce01382bd23a156140ba1d3700f7171292a77 just now using a working SecureDrop install. Here is what I did:
 1) Manually install SecureDrop and the firewall
 2) Factory reset the pfSense firewall router
 3) Using the clearnet web browser (Unsafe browser) I went to my branch and downloaded the `.xml` files as "raw" from GitHub (**note**: this won't work using Tor due to the `clearnet` user chroot in Tails).
 4) Confirmed the factory reset was successful, no firewall rules were in the pfSense admin GUI. Also the admin user's password is reset. 
  5)  Follow the steps in our [docs](https://docs.securedrop.org/en/stable/network_firewall.html) up to "SecureDrop Configuration"
 6) Restore the `xml` files for each section.
 7) Reboot the firewall (important step)
 8) Test connectivity to the `.onion` using both Tor Browser and ssh.

I will test further using a fresh SD install next week.
